internal/docker/builder.go

```diff
@@ -124,16 +124,16 @@ func (d *Builder) removeImages() error {
 			d.log("Not cleaning up image with tags: %v", img.RepoTags)
 			continue
 		}
-		bprintName := img.Labels["complement_blueprint"]
+		contextStr := img.Labels[complementLabel]
```

Comment:
Re-using the `contextStr` because it already packs in the blueprint name and homeserver name like `fed.perf_many_messages.hs1`